bazel/packaging/bin_wrapper/BUILD

```diff
@@ -17,7 +17,6 @@ go_binary(
     visibility = ["//bazel/packaging:__pkg__"],
     x_defs = {
         "binaryPath": "/opt/redpanda/libexec/redpanda",
-        "ldLibraryPath": "/opt/redpanda/lib",
```

Comment:
Do we even need these anymore with this approach? Should we remove the wrappers and just put the real binaries in bin/ instead of libexec/?